brian_cheung/week_two_challenges/kanye_records.md

```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |hash_iterate| puts hash_iterate[:title] end
 ```
 
 2. Print the year every Kanye record was released.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |hash_iterate| puts hash_iterate[:year] end
 ```
 
 3. Look into the future and add a forthcoming Kanye record into our array. It should be released in 2021 and have a title that is completely asinine.
 
 ```ruby
-# write your code here
+kanyeRecords << {year: 2021, title: "Tis the Season to dress it salty"}
 ```
 
 4. Iterate through Kanye's record collection and print out the titles of all the records released after Kanye's ego exploded (this happened in 2007).
 
 ```ruby
-# write your code here
+kanyeRecords.each do |i|
+  if i[:year] >= 2007
+    puts i[:title]
+  end
+end
 ```
 
 5. Iterate through the collection. For each record, print out a sentence detailing each record (e.g. "'Graduation' was released in 2007.").
 
 ```ruby
-# write your code here
+kanyeRecords.each do |i|
+  puts "'#{i[:title]} was released in #{i[:year]}"
+end
 ```
 
 6. Iterate through collection. add a key value pair to each record with the key as `label` and the value as `"def jam"`.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |i|
+  i['label'] = "def jam"
+end
 ```
```

Comment:
Remember, be consistent within your data structure. You're using symbols for `year` and `title`, so you should use a symbol for the `label` as well.